generator/common/beatgen/setup/setup.go

```diff
@@ -48,17 +48,7 @@ func RunSetup() error {
 	if err != nil {
 		return errors.Wrapf(err, "error copying pkg to %s", vendorPath)
 	}
-	return LinkImportsHelper()
-}
-
-// LinkImportsHelper links generate_imports_helper.py
-func LinkImportsHelper() error {
-	vendorPath := "./vendor/github.com/"
-	pwd, err := os.Getwd()
-	if err != nil {
-		return errors.Wrap(err, "error gettting current directory")
-	}
-	return sh.Run("ln", "-sf", filepath.Join(pwd, vendorPath, "elastic/beats/metricbeat/scripts/generate_imports_helper.py"), filepath.Join(pwd, vendorPath, "elastic/beats/script/generate_imports_helper.py"))
```

Comment:
This is sth which requires a careful look, not sure if it's ok to remove. Maybe @fearful-symmetry can comment here.

Reply:
The new go generator has its own code for linking that, so I'm not sure what's using this? 
https://github.com/elastic/beats/blob/72f2bf96b2465b277579253b24cb0d42ff1e5538/generator/common/beatgen/setup/setup.go#L55-L62